### DISCUSSION TITLE
Videos with large number of comments?

### DISCUSSION THREAD

**Discussion by BergQuester** [ThioJoe/YT-Spammer-Purge#610] [Q&A]
I was looking for videos with large numbers of comments to target as fertile spam grounds and came across this list of videos with over 1 million comments: https://youtube.fandom.com/wiki/Most-Commented_YouTube_Videos 

The top video, with over 19 million comments is located at https://www.youtube.com/watch?v=9P5FA3Em1P8

With the quota limiting to about 10,000 reports/day, how might scanning and reporting work with videos with such large amounts of comments?

**Comment by Firecul** [comment]
Short answer, it wouldn't afaik. I don't think you can get all of the 19 million comments with a 10,000 requests quota. I might be wrong obviously.

**Comment by ThioJoe** [comment]
With a 10k quota you'd be able to scan about a million comments per day. So yea there's basically no way to scan that many.

**Reply by BergQuester** [reply]
Some sort of “large job” mode for chopping large jobs such as videos/channels with lots of comments or channels with lots of videos would be beneficial.

**Comment by hunggiadao** [comment]
Is there a way to stop the scan when you've reached the 10k quota for spam comments? That way even though you can't go through all the comments you are still able to report the spams you have detected thus far.

**Reply by KendallDoesCoding** [reply]
Yes, it gives a messege:

"Error occured, quota is over"  or something like that and if you press enter it will show the spam comments detected, only issue is that if your quota is up you cant delete or report comments, so you just wasting your time, so maybe it should show the messege saying quota is up when you've used 9k to 9.5k of the quota.

**Comment by TechStudent10** [comment]
I thought about this while I was scanning Me at the zoo and hopefully get rid of some of those "Sub4Sub" and scam comments, or at least report them. It took forever and I had to reboot my computer so yeah.
